packages/server/graphql/mutations/addReactjiToReactable.ts

```diff
@@ -164,13 +165,16 @@ const addReactjiToReactable = {
 
     const data = {reactableId, reactableType}
     const {meetingType} = await dataLoader.get('newMeetings').load(meetingId)
+    const isAIComment = reactable instanceof Comment && reactable.createdBy === PARABOL_AI_USER_ID
```

Comment:
-1 Does not work for me:
![Screenshot from 2023-01-09 17-19-49](https://user-images.githubusercontent.com/7331043/211356224-74b3e84e-914f-4cb9-b2dc-fa6f40f8af0d.png)
![Screenshot from 2023-01-09 17-20-03](https://user-images.githubusercontent.com/7331043/211356237-2c62f5ec-0fd4-4f60-ae09-a9030828f85a.png)